packages/webcomponents/src/components/order-terminals/order-terminals.tsx

```diff
@@ -39,6 +40,8 @@ export class OrderTerminals {
   @State() order: TerminalOrder;
   @State() totalQuantity: number = 0;
 
+  addressHeader = this.shipping ? 'Shipping Address: ' : "Business Address: ";
```

Comment:
Can we make the quotes consistent (single vs double) and also I think we can remove the space at the end of each heading